### ISSUE TITLE
documentInternalElements is not honoured for interfaces

### ISSUE THREAD

**Issue by RobSiklos** [DotNetAnalyzers/StyleCopAnalyzers#3095]
It seems that the `documentInternalElements` setting which we can specify in `stylecop.json` is completely ignored when it comes to `interface` types and their members.

Tested with 1.2.0-beta.113

### Repro
`stylecop.config`:
```json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "documentInternalElements": false
    }
  }
}
```
C#:
```csharp
// No warning generated, as expected
internal class Foo
{
    // No warning generated, as expected
    public void M() { }
}

// Warning SA1600 generated for IFalsePositive - NOT EXPECTED
internal interface IFalsePositive
{
    // Warning SA1604 generated for Foo() - NOT EXPECTED
    /// <exception cref="ArgumentException">You done messed up A-A-Ron.</exception>
    void Foo();
}

```

**Comment by sharwell** [comment]
Duplicate of #2934
